ext_mongodb.php

```diff
@@ -354,23 +357,34 @@ public function __construct(mixed $filter, array $options = array())
 			$this->query['flags'] = array_key_exists('flags', $options ) ? (int) $options['flags'] : 0;
 			$this->query['limit'] = array_key_exists('limit', $options ) ? (int) $options['limit'] : 0;
 			$this->query['skip'] = array_key_exists('skip', $options ) ? (int) $options['skip'] : 0;
-		}
 
-		if (array_key_exists('modifiers', $options)) {
-			Utils::mustBeArrayOrObject('modifiers', $options['modifiers']);
-			foreach ($options['modifiers'] as $key => $value) {
-				$this->query['query'][$key] = $value;
+			if (array_key_exists('readConcern', $options)) {
+				if (!($options['readConcern'] instanceof \MongoDB\Driver\readConcern)) {
+					throw new \MongoDB\Driver\Exception\InvalidArgumentException(
+						'Expected "readConcern" option to be MongoDB\Driver\ReadConcern, ' .
+						gettype($options['readConcern']) . ' given'
+					);
+				} else {
+					$this->query['readConcern'] = $options['readConcern']->getLevel();
```

Comment:
From this point forward, is `$this->query` purely internal, or could it be revealed to the user? I'm wondering if debugging a Query might end up printing "readConcern" as a string value instead of the object.

Also, this just made me realize that PHPC's debug handler doesn't print the read concern at all. It's only ever stored on the internal struct and I neglected to add it to `php_phongo_query_get_debug_info()`.